.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```

Comment:
I'm on a Windows machine at the moment, adding this keeps CRLF out of the repo...